deploy/helm/node-collector-config.yaml

```diff
@@ -75,6 +75,12 @@ processors:
     transforms:
       {{- include "common-config.metricstransform-preprocessing-cadvisor" . | nindent 6 }}
 
+  filter/preprocessing-cadvisor:
+    error_mode: ignore
+    metrics:
+      datapoint:
+        - 'metric.name == "k8s.container.status" and value_double != 1'
```

Comment:
Why? Metric `k8s.container.status` is created only in the metrics-collector. It does not exist in the node-collector, so there is nothing to filter.

Reply:
Ah, I thought it is coming from cAdvisor which can be scraped on both. Ok removed from node-collector and merged `filter/preprocessing-cadvisor` with `filter/preprocessing`